File: tkpddigital/src/main/java/com/tokopedia/digital/cart/data/entity/response/AdditionalInfo.java
package com.tokopedia.digital.cart.data.entity.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang3.builder.ToStringBuilder;

import java.util.List;

/**
 * @author anggaprasetiyo on 2/27/17.
 */

public class AdditionalInfo {
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("detail")
    @Expose
    private List<Detail> detail = null;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public List<Detail> getDetail() {
        return detail;
    }

    public void setDetail(List<Detail> detail) {
        this.detail = detail;
    }

    @Override
    public String toString() {
        return ToStringBuilder.reflectionToString(this);
    }

}

File: features/recharge/digitalproduct/src/main/java/com/tokopedia/digital/newcart/data/entity/response/cart/AdditionalInfo.kt
package com.tokopedia.digital.newcart.data.entity.response.cart

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author anggaprasetiyo on 2/27/17.
 */

class AdditionalInfo {
    @SerializedName("title")
    @Expose
    var title: String? = null
    @SerializedName("detail")
    @Expose
    var detail: List<Detail>? = null
}